vortex-array/src/array/sparse/compute.rs

```diff
@@ -46,6 +46,7 @@ impl AsContiguousFn for SparseArray {
                     .collect_vec(),
             )?,
             arrays.iter().map(|a| a.len()).sum(),
+            Scalar::Null(NullScalar::new()),
```

Comment:
I think you need to assert that all array fill_values are the same, and then take self.fill_value()